### PR TITLE
Load KenLM with LAZY

### LINKED ISSUE #2384: Investigate long loading time of LM
On some arch (RPi, Android) we can feel that there's a several-seconds delay when enabling the language model. This is only on first run. LM should already be `mmap()`'d, so we should investigate what happens, this cripples user experience.

### PR DESCRIPTION
**Pull request by lissyx**
Fixes #2384

### CODE REVIEW

File: native_client/ctcdecode/scorer.cpp [hunk 1]
```diff
@@ -83,7 +83,7 @@ void Scorer::setup(const std::string& lm_path, const std::string& trie_path)
   if (!has_trie) { // no trie was specified, build it now
     RetrieveStrEnumerateVocab enumerate;
     config.enumerate_vocab = &enumerate;
-    language_model_.reset(lm::ngram::LoadVirtual(filename, config));
+    language_model_.reset(new lm::ngram::QuantArrayTrieModel(filename, config));
```
Comment (reuben) [inline native_client/ctcdecode/scorer.cpp]: Why switch back to static model type? Was there significant overhead in `LoadVirtual`?

Comment (lissyx) [reply native_client/ctcdecode/scorer.cpp]: Doc in the KenLM says we should avoid using virtual load at all expense.

Comment (reuben) [reply native_client/ctcdecode/scorer.cpp]: I know, but did it actually hurt performance? It's nice to be able to experiment with different data structures without having to change the code and recompile.

Comment (lissyx) [reply native_client/ctcdecode/scorer.cpp]: I have not measured accurately, but I don't think it did. I'll recheck that.

**Comment by lissyx** [comment]
r=me on irc

**Comment by reuben** [comment]
Corresponding issue: #2384.

**Pull request merged**